plugins/uefi-dbx/README.md

```diff
@@ -4,5 +4,33 @@ UEFI dbx Support
 Introduction
 ------------
 
-This plugin checks if the UEFI dbx contains all the most recent revoked
+Updating the UEFI dbx prevents starting EFI binaries with known security issues,
```

Comment:
probably should mention what dbx actually is 🗡️

Reply:
I used `revocation database` again here, thanks.